play/PlayerRules/CharacterCreation/03_Feats.txt

```diff
@@ -208,6 +208,15 @@ Prerequisites: None
 Somehow you always aim at the right location, at the right time. 
 +10% Crit chance, +5% Extra-Crit chance.
 
+Punch Drunk:
+Prerequisite: Fort > 3
+Take a +5% to miss on all attacks when not drunk. 
+Instead of taking a penalty to accuracy when using melee attacks when 
+intoxicated, you get a -5% to miss when buzzed and a -15% to miss when
+hammered. These bonuses only apply to melee attacks. You also get a 
++10% on rolls to avoid blacking out.  
+

```

Comment:
That sounds fine to me; my complaint is on the mechanical level. Taking this feat kind of forces you to play at a disadvantage, and that seems self defeating. The feat effectively gives you a -1 to perception for ranged fighting while not drunk, and being drunk already gives you a -1 perception. And your stated target was for the relevant player to be buzzed, which only yields a +5% melee accuracy while buzzed with the feat. Oh, and I'm not sure if this is intentional, but the feat only states that you no longer have maluses to melee attacking with the buzzed/hammered, but it doesn't say anything about ranged attacks.